.circleci/config.yml

```diff
@@ -783,7 +783,7 @@ jobs:
                       - v0.4-torch-{{ checksum "setup.py" }}
                       - v0.4-{{ checksum "setup.py" }}
             - run: pip install --upgrade pip
-            - run: pip install .[torch,testing,sentencepiece,onnxruntime]
+            - run: pip install .[torch,testing,sentencepiece,onnxruntime,vision]
```

Comment:
I've added the vision dependency here to the CI tests. I couldn't see any other files in `.github/workflows` where I needed to add it, but please let me know otherwise